CSC307Fall2018/team-too
# Unique user emails & "Foxycle" & Resolve double create

# Summary
- Updated the data models such that user emails must now be unique
- Fixed the typo in the Header

# QA
- Make sure you cannot create a user (via `createUser` mutation) with an email that already exists in the database
- verify that the header is properly titled
